### PR TITLE
Update ThetaLinesTransformer's docstring

### PR DESCRIPTION
**Pull request by GuzalBulatova**
#### Reference Issues/PRs
References #1245.

#### What does this implement/fix? Explain your changes.

Updates the docstring for it to satisfy the documentation requirements.

#### Does your contribution introduce a new dependency? If yes, which one?

No

#### What should a reviewer concentrate their feedback on?

Are there other sections that can be added?

### CODE REVIEW

**Review by mloning — APPROVED** [review]

File: sktime/transformations/series/theta.py [hunk 1]
```diff
@@ -18,13 +18,39 @@
...
+    theta : list or tuple of int or float, default=(0,2)
+        Theta-coefficients to use in transformation.
+
     Notes
     -----
-    Implements decomposition as described in [1]_.
+    Depending on the value of the Theta-coefficient, Theta-lines either augment the
+    long-term trend (0 < Theta < 1) or the the short-term behaviour (Theta > 1).
+    Special cases:
+    Theta == 0 : deflates input data to linear trend;
```
Comment (mloning) [inline sktime/transformations/series/theta.py:44]: I'd make this a list of bullet points.

File: sktime/transformations/series/theta.py [hunk 2]
```diff
@@ -18,13 +18,39 @@
 class ThetaLinesTransformer(_SeriesToSeriesTransformer):
     """Decompose the original data into two or more Theta-lines.
 
+    Implementation of decomposition for Theta-method [1]_ as described in [2]_.
+
+    Overview: Input :term:`univariate series <Univariate time series>` of length
+    "n" and ThetaLinesTransformer modifies the local curvature of the time series
+    using Theta-coefficients passed through the parameter `theta`.
+
+    The pimary parameter:
```
Comment (mloning) [inline sktime/transformations/series/theta.py:27]: Instead of making this a bullet point, I'd rephrase it into a sentence, it looks a bit strange on the website.

**Review by mloning — COMMENTED** [review]
Just had a look here: https://sktime--1312.org.readthedocs.build/en/1312/api_reference/auto_generated/sktime.transformations.series.theta.ThetaLinesTransformer.html#sktime.transformations.series.theta.ThetaLinesTransformer and added a few comments to improve it further

File: sktime/transformations/series/theta.py [hunk 3]
```diff
@@ -1,9 +1,9 @@
-#!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
+# !/usr/bin/env python3 -u
 # License: copyright: sktime developers, BSD-3-Clause License (see LICENSE file).
-"""Implements Theta-lines transformation for use with automatic theta forecasting."""
+"""Implements Theta-lines transformation for use with automatic Theta forecasting."""
```
[the remark this answers is not in the document]
Comment (GuzalBulatova) [reply sktime/transformations/series/theta.py:4]: Thanks! Yes, you're right Lovkush, I'll make the change. 👍

**Review by mloning — APPROVED** [review]

**Review by Lovkush-A — APPROVED** [review]
suggested changes were all implemented. full steam ahead!

**Review by Lovkush-A — APPROVED** [review]
I'm going to approve this again, despite the tests not being complete. Looks like it is sktime wide issue, not some about this PR.
